### ISSUE TITLE
A reference from w3c

### ISSUE THREAD

**Issue by mor10** [mor10/a11y-dropdown#1]
https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html

**Comment by mor10** [comment]
https://www.w3.org/TR/wai-aria-practices-1.1/#menu

**Comment by missmatsuko** [comment]
I was going to post this. I think the demo is missing a lot of the features from keyboard navigation guidelines, especially using arrows to navigate.
